bitmap-team-deuces/test/transform-test.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
```

Comment:
You've all done a great job creating unit tests that test every part of your lib folder!!!  :D